PowershellTools.Common/ServiceManagement/PowershellHostServiceExceptionDetails.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PowerShellTools.Common.ServiceManagement
+{
+    [DataContract]
+    class PowershellHostServiceExceptionDetails
+    {
+        public static readonly PowershellHostServiceExceptionDetails Default = new PowershellHostServiceExceptionDetails();
+
+        [DataMember]
+        public String Message { get; private set; }
+
+        public PowershellHostServiceExceptionDetails()
+        {
+            this.Message = "There is a problem in the powershell host service.";

```

Comment:
So far we don't show it to end user, just for internal logging purpose. However we may need look over the entire project to see if there is anything needs to be localized, maybe March? Anyway I'd like to enter a task for localization work, if there is not yet one.